common/djangoapps/student/management/commands/assigngroups.py

```diff
@@ -82,9 +82,11 @@ def handle(self, *args, **options):
             v = random.uniform(0, 1)
             group = group_from_value(groups, v)
             group_objects[group].users.add(user)
-            f.write("Assigned user {name} ({id}) to {group}\n".format(name=user.username,
-                                                                      id=user.id,
-                                                                      group=group))
+            f.write(u"Assigned user {name} ({id}) to {group}\n".format(
```

Comment:
For log messages, please use [lazy logging](http://stackoverflow.com/a/4149231/2669626)